src/location.rs

```diff
@@ -11,29 +11,18 @@ use crate::xor_space::{Prefix, XorName};
 /// Source location
 #[derive(Serialize, Deserialize, PartialEq, PartialOrd, Eq, Ord, Clone, Copy, Hash, Debug)]
 pub enum SrcLocation {
-    /// A single section whose prefix matches the given name
-    Section(XorName),
-    /// A set of nodes with names sharing a common prefix - may span multiple `Section`s present in
-    /// the routing table or only a part of a `Section`
-    PrefixSection(Prefix<XorName>),
-    /// A single node
+    /// A single node with the given name.
     Node(XorName),
+    /// A single section with the given prefix.
+    Section(Prefix<XorName>),
```

Comment:
Not 100% sure since SrcLocation::Section means something diferent from DstLocation::Section, seem a bit strange.

Reply:
Should I rename it to `Prefix` perhaps?

Reply:
Maybe, but we might loose the context that this is a Section?

Reply:
Maybe the fact that they contain different type but named the same is less confusing. (like you did).

Reply:
So `PrefixSection` then? As it was?

Reply:
It was meaning a different thing and is not particularly clear. I'd say lets forget that comment for now, I'm not sure the "fix" we have are helping.